projects/aca-shared/src/lib/directives/contextmenu/contextmenu.directive.ts

```diff
@@ -48,7 +48,7 @@ export class ContextActionsDirective implements OnInit, OnDestroy {
 
       if (this.enabled) {
         const target = this.getTarget(event);
-        if (target) {
+        if (target && !target.classList.contains('adf-no-content-container')) {
```

Comment:
This is coupling the directive with a specific class... 
In my opinion the parent component in case of no content shouldn't add the capability of right click

Reply:
I have added a seperate method in case datatable is empty